explorer/nodewatch/tests/test_HeightChartBuilder.py

```diff
@@ -1,3 +1,6 @@
+# pylint: disable=invalid-name
```

Comment:
Not sure why this doesnt fail for all run though 🤔 
https://jenkins.symboldev.com/job/Symbol/job/generated/job/product/job/nodewatch/job/dev/